catalog/app/components/PaymentDialog/messages.js

```diff
@@ -4,24 +4,28 @@ import { defineMessages } from 'react-intl';
 export default defineMessages({
   already: {
     id: 'app.containers.Profile.alreadyFree',
-    defaultMessage: 'Already on plan',
+    defaultMessage: 'Nothing to do',
   },
   free: {
     id: 'app.containers.Profile.free',
     defaultMessage: 'Free',
   },
-  business_monthly_490: {
-    id: 'app.containers.Profile.business_monthly_490',
-    defaultMessage: 'Business',
-  },
-  business_member: {
-    id: 'app.containers.Profile.business_member',
-    defaultMessage: 'Business',
-  },
   individual_monthly_7: {
     id: 'app.containers.Profile.individual_monthly_7',
     defaultMessage: 'Individual',
   },
+  team_monthly_490: {
+    id: 'app.containers.Profile.team_monthly_490',
+    defaultMessage: 'Team',
+  },
+  team_unpaid: {
+    id: 'app.containers.Profile.team_unpaid',
+    defaultMessage: 'Team (30-day trial)',
```

Comment:
Hm, Stripe has built-in support for trials. We could just use it, and not even have the "unpaid" plan. I only did it because the paid plan currently requires a credit card.

Reply:
hard to depend on stripe; e.g. see the logic i do with `plan.rank`, etc. so unless this is demonstrably fragile i'm hoping to keep a client-side object of all possible plans and meta-data.